ui/tests/acceptance/secrets/backend/database/secret-test.js

```diff
@@ -574,7 +574,7 @@ module('Acceptance | secrets/database/*', function (hooks) {
     await rolePage.name('bar');
     assert
       .dom('[data-test-component="empty-state"]')
-      .exists({ count: 2 }, 'Two empty states exist before selections made');
```

Comment:
Can we keep this assertion and move it down until after a database is selected? The message could be updated to something like "two empty states show after a database is selected"